app/assets/stylesheets/modules/_landing.scss

```diff
@@ -2,9 +2,23 @@
   background-color: $white;
 
   &-title {
+    background: image-url("projectgiveback.png");
+    background-repeat: no-repeat;
+    background-size: em(140);
+    display: block;
```

Comment:
Properties should be ordered background, background-repeat, background-size, color, display, height, margin, margin-bottom, text-align, width